README.md

```diff
@@ -81,6 +56,14 @@ We welcome all contributions of code and translations. Please feel free to fork
     ```
   - This could take up to about a minute depending on your internet connection.
 
+## Translations
+If you would like to assist with the NamelessMC development by providing an updated translation, please feel free to fork the repository here on GitHub and create any pull requests. Or, if you don't want to deal with creating a pull request, please send us a zip file and we'll add it to the repository for you. 
+Just make sure you translate the [latest development version](https://github.com/NamelessMC/Nameless/archive/refs/heads/v2.zip), not the latest release or we won't be able to merge your translations! 
+Currently other available translations can be found [here](https://translate.namelessmc.com). Please note, not all translations may be up to date. To discuss changes with fellow translators, visit the [NamelessMC Translators](https://discord.gg/7Dku3fE) discord server.
+NamelessMC translations are kindly provided by the community. 
+Currently available translations can be found [here](https://github.com/NamelessMC/Nameless/tree/v2/custom/languages). Please note, not all translations may be up to date.
+
 ## Special Thanks
-- All NamelessMC [contributors](https://github.com/NamelessMC/Nameless/graphs/contributors).
-- [JetBrains](https://www.jetbrains.com/), whose products are used to develop the NamelessMC project.
+* All [NamelessMC Translation contributors](https://github.com/NamelessMC/Nameless/CONTRIBUTORS.md).
```

Comment:
Nitpick: I would swap the order of translation contributors and contributors links